src/components/MaximWPproj2/CentCont.styled.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+
+import { ButtonStyle, CentreCont, Detailsincont, Hrstyle2, StyledHeader, StyledNumber } from './styles/Container.styled'
+
+type Props = {
+  title: string
+  number: string
+  text1: string
+  text2: string
+  text3: string
+}
+
+export const CentCont = (props: Props) => {
+  return (
+    <CentreCont>
+      <StyledHeader containertype={'Main'}>{props.title}</StyledHeader>
+      <StyledNumber containertype={'Main'}>{props.number}</StyledNumber>
+      <Detailsincont containertype={'Main'}>
+        <Hrstyle2></Hrstyle2>
+        {props.text1}
+        <Hrstyle2></Hrstyle2>
+        {props.text2}
+        <Hrstyle2></Hrstyle2>
+        {props.text3}
+      </Detailsincont>
+      <ButtonStyle> Learn more </ButtonStyle>
```

Comment:
these are styled components so try to name them as Styled.. instead of ..Style (only use this for when its a style you are exporting) eg 
```
//in styles file
export const ButtonStyle = { //your styles }

//to use it
<button style={ButtonStyle}>...
```